php/route_guide/README.md

```diff
@@ -0,0 +1,259 @@
+#gRPC Basics: PHP
+
+This tutorial provides a basic PHP programmer's introduction to working with gRPC. By walking through this example you'll learn how to:
+
+- Define a service in a .proto file.
+- Generate client code using the protocol buffer compiler.
+- Use the PHP gRPC API to write a simple client for your service.
+
+It assumes a passing familiarity with [protocol buffers](https://developers.google.com/protocol-buffers/docs/overview). Note that the example in this tutorial uses the proto2 version of the protocol buffers language.
+
```

Comment:
Maybe also add something along the lines of: "Also note that currently you can only create clients in PHP for gRPC services - you can find out how to create gRPC servers in our other tutorials."

(do we ever envisage PHP servers?)